README.md

```diff
@@ -1,138 +1,100 @@
-
-[Go To Documentation](https://chakra-ui.com/docs/getting-started)
-
-### **CSS / styled-jsx**
-
-Next.js comes with built-in support for CSS and styled-jsx. Styled-jsx is full, scoped and component-friendly CSS support for JSX (rendered on the server or the client).
-
-[Go To Documentation](https://github.com/vercel/styled-jsx)
-
-### **Fetch**
-
-Next.js has a built-in polyfill for the fetch API. You don&#39;t need to worry about using it on either server or client side.
-
-[Go To Documentation](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)
-
-### **Storybook**
-
-Storybook is an open source tool for developing UI components in isolation for React, Vue, Angular, and more. It makes building stunning UIs organized and efficient.
-
-[Go To Documentation](https://storybook.js.org/docs/react/get-started/introduction)
-
-### **SVGR**
-
-Transform SVGs into React components.
+## Development
 
-[Go To Documentation](https://react-svgr.com/docs/getting-started/)
+Linters run automatically as a pre-commit hook on edited ```.js, .jsx, .ts, .tsx``` files you edit and commit
 
-### **Environment Variables**
-
-Use environment variables in your next.js project for server side, client or both.
-
-[Go To Documentation](https://github.com/vercel/next.js/tree/canary/examples/environment-variables)
-
-### **SWR**
-
-React Hooks library for data fetching from Vercel
-
-[Go To Documentation](https://swr.vercel.app/)
-
-### **Unstated-Next**
-
-Unstated-Next will be used for state management
-
-[Go To Documentation](https://github.com/jamiebuilds/unstated-next)
-
-### **next-i18next**
-
-next-i18next is a plugin for Next.js projects that allows you to get translations up and running quickly and easily, while fully supporting SSR, multiple namespaces with codesplitting, etc.
-
-[Go To Documentation](https://github.com/isaachinman/next-i18next)
-
-### **ESLint**
-
-A pluggable and configurable linter tool for identifying and reporting on patterns in JavaScript. Maintain your code quality with ease.
-
-[Go To Documentation](https://eslint.org/docs/user-guide/getting-started)
-
-### **Prettier**
-
-An opinionated code formatter; Supports many languages; Integrates with most editors.
-
-[Go To Documentation](https://prettier.io/docs/en/index.html)
-
-### **lint-staged**
-
-The concept of lint-staged is to run configured linter (or other) tasks on files that are staged in git.
-
-[Go To Documentation](https://github.com/okonet/lint-staged)
-
-### **Testing Library**
-
-The React Testing Library is a very light-weight solution for testing React components. It provides light utility functions on top of react-dom and react-dom/test-utils.
-
-[Go To Documentation](https://testing-library.com/docs/)
-
-### **Cypress**
-
-Fast, easy and reliable testing for anything that runs in a browser.
+To run the linters manually:
+```bash
+# Runs linting
+yarn linting
```

Comment:
I believe this should be `yarn lint`. I get an error if I run `yarn linting`.

Reply:
Good catch